apps/chat/src/utils/creditPeriods.ts

```
@@ -0,0 +1,168 @@
+  const now = new Date()
+
+  switch (resetPeriod) {
+    case ResetPeriod.DAILY:
+      // Start of current day in UTC
+      return new Date(
+        Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate())
+      )
+
+    case ResetPeriod.WEEKLY:
+      // Start of current week (Monday 00:00 UTC)
+      const startOfWeek = new Date(now)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)